.github/workflows/pytest.yml

```
@@ -4,11 +4,9 @@ on:
   push:
     branches:
       - main
-      - develop
   pull_request:
     branches:
       - main
```

This should allow this to run on all PRs.

```suggestion
  pull_request:
```

---

Didn't realize that was the syntax for it.  Tests are now running properly locally so I'll see if applying this change works with GHA.

---

BTW, I'm not a docker expert but I believe that "docker run" will create a new container and that is why the previous command doesn't persist to the next one. Maybe try to do everything in a single call?

```
docker run -u root glider-dac-build /bin/bash -c "pip install --no-cache -r /glider-dac/test_requirements.txt && pytest /glider-dac/glider_dac/tests"
```